lib/sphinx-preview.js

```diff
@@ -3,6 +3,8 @@
 import { CompositeDisposable } from 'atom';
 import Docker from 'dockerode';
 
+const os = require('os');
```

Comment:
Maybe use import instead of require to be consistent
```suggestion
import { os } from 'os';
```

Reply:
see [55133fb](https://github.com/dldl/sphinx-preview/pull/16/commits/55133fbc4fe7ed39909540cdf9a0caf85caec077)